RussianFilter/sections/specific.txt

```diff
@@ -3856,7 +3856,7 @@ mel.fm##.i-layout_branding > .i-layout__branding-link_a
 referatwork.ru##.ie_container > div[class="extra"][style="padding-right: 20px;"]
 wikianime.ru##.ifiwerearichman-bg-clickable
 myzuka.me##.iframediv.center[style]
-evekino.com,games-full.info##.ignore-select
+thelifecoshop.com,evekino.com,games-full.info##.ignore-select
```

Comment:
`thelifecoshop.com` why is this domain here?
Please add the `manshet.org#?#.card-title ~ div.ignore-select div.subscr-title:upward(2)` to the Annoyances filter -> `annoyances.txt` near `!# Subscriptions/newsletters (not popups)`